src/build.py

```diff
@@ -965,6 +965,36 @@ def V8():
     CopyBinaryToArchive(os.path.join(V8_OUT_DIR, a))
 
 
+def Jsvu():
+  buildbot.Step('jsvu')
+
+  try:
+    if IsWindows():
+      # jsvu OS identifiers:
+      # https://github.com/GoogleChromeLabs/jsvu#supported-engines
+      os_id = 'windows64'
+      js_engines = 'chakra'
+    elif IsMac():
+      os_id = 'mac64'
+      js_engines = 'javascriptcore'
+    else:
+      return
+
+    # https://github.com/GoogleChromeLabs/jsvu#installation
+    proc.check_call(['npm', 'install', 'jsvu', '-g'])
+
+    # https://github.com/GoogleChromeLabs/jsvu#integration-with-non-interactive-environments
+    proc.check_call(['jsvu',
```

Comment:
likewise here. use an explicit path to JSVU.

Reply:
Actually is `jsvu` a binary or a shell script, or a JS file that we have to run with node, or what?

Reply:
Done. Note that `npm install -g` puts `jsvu` [wherever Node is installed](https://docs.npmjs.com/files/folders#tldr).